client/src/components/Monument/SuggestChanges/SuggestChanges.js

```diff
@@ -17,7 +24,7 @@ export default class SuggestChanges extends React.Component {
                 <Card.Body>
                     <p>We pride ourselves on keeping up to date and accurate information.</p>
                     <p>If you think something on this page is incorrect or outdated, please suggest a change. If your change is approved we'll send you an email to let you know, and you will be listed as a contributor on this page!</p>
-                    <Button variant="primary">SUGGEST A CHANGE</Button>
+                    <Button variant="primary" onClick={() => this.handleButtonClick()}>SUGGEST A CHANGE</Button>
```

Comment:
You can just make this a link instead of passing callbacks all the way down. You will need to pass the `monument` down though
```<Link className="btn btn-primary" href={`/update-monument/${monument.id}`}>SUGGEST A CHANGE</Link>```

Reply:
I don't think we can do that anymore now that the button links to different pages based on if its on the Home page or the View page.

Reply:
Ok. In the long run I would prefer to have it be a `Link` because it allows you to right click and open in new tab whereas buttons do not. But for the purposes of getting things ready for user testing I don't think it needs to be fixed right away